src/iotjs_binding.c

```diff
@@ -265,44 +261,17 @@ jerry_value_t iotjs_jval_get_property_by_index(jerry_value_t jarr,
 }
 
 
-#ifndef NDEBUG
-static jerry_value_t iotjs_jargs_get(const iotjs_jargs_t* jargs,
-                                     uint16_t index) {
-  const IOTJS_VALIDATED_STRUCT_METHOD(iotjs_jargs_t, jargs);
-
-  IOTJS_ASSERT(index < _this->argc);
-  return _this->argv[index];
-}
-#endif
-
-
 jerry_value_t iotjs_jhelper_call(jerry_value_t jfunc, jerry_value_t jthis,
                                  const iotjs_jargs_t* jargs) {
   IOTJS_ASSERT(jerry_value_is_object(jfunc));
 
   jerry_value_t* jargv_ = NULL;
   jerry_length_t jargc_ = iotjs_jargs_length(jargs);
 
-#ifdef NDEBUG
-  jargv_ = (jerry_value_t*)jargs->unsafe.argv;
-#else
-  if (jargc_ > 0) {
-    unsigned buffer_size = sizeof(jerry_value_t) * jargc_;
-    jargv_ = (jerry_value_t*)iotjs_buffer_allocate(buffer_size);
-    for (unsigned i = 0; i < jargc_; ++i) {
-      jargv_[i] = iotjs_jargs_get(jargs, i);
-    }
-  }
-#endif
+  jargv_ = (jerry_value_t*)jargs->argv;
 
   jerry_value_t jres = jerry_call_function(jfunc, jthis, jargv_, jargc_);
```

Comment:
`jargv_` is not needed anymore. You can simply write:
`jerry_value_t jres = jerry_call_function(jfunc, jthis, jargs->argv, jargc_);`

Reply:
Indeed, you are right! I'll fix this.